documentation/source/coroutines.rst

```diff
@@ -47,8 +51,25 @@ Coroutines can :keyword:`return` a value, so that they can be used by other coro
         if first == second:
             raise TestFailure("Signal did not change")
 
+Concurrent Execution
+====================
+
+Coroutines can be scheduled for concurrent execution with :func:`~cocotb.fork`, :func:`~cocotb.start`, and :func:`~cocotb.start_soon`.
+
+.. note::
+    :func:`~cocotb.start` and :func:`~cocotb.start_soon` are new scheduling functions.
```

Comment:
The "new" here will age fast, maybe better express it with another `.. versionadded::` in the note?